github.js

```diff
@@ -178,6 +178,12 @@
          this.orgs = function(cb) {
             _request('GET', '/user/orgs', null, cb);
          };
+         
+         // List members of given organization
+         // -------
+         this.orgMembers = function (orgId, cb) {
```

Comment:
I think that having an API that expect an ID isn't very easy to use. As we do for other methods I think we should receive a name and do the "dirty" work on behalf of the user (retrieving the ID by the name).